apps/web/src/ee/clerk/components/QuestionnaireForm.tsx

```diff
@@ -43,51 +28,74 @@ export function QuestionnaireForm() {
     handleSubmit,
     formState: { errors },
     control,
-  } = useForm<UpdateExternalOrganizationDto>({});
+  } = useForm<IOrganizationUpdateForm>({});
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏ didn't we have a push to remove the `I` from our interface names?

Reply:
I think it mainly on API due to the SDK autogeneration from swagger, but good point I changed it here as well to be consistent.